WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -3,12 +3,14 @@ import UIKit
 import RxSwift
 import WordPressComAnalytics
 
-func AccountSettingsViewController(account account: WPAccount) -> ImmuTableViewController {
-    let service = AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+func AccountSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    let service = account.map({ account in
+        return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+    })
```

Comment:
If the account is nil, service will be nil, which is an acceptable scenario. In this case I like map more to handle the optionals, as it more concise. The alternative would be something like this

```swift
var service: AccountSettingsService? = nil
if let account = account {
  service = AccountSettingsService(...)
}
```